api/src/main/java/bio/terra/tanagra/service/query/EntityDataset.java

```diff
@@ -0,0 +1,58 @@
+package bio.terra.tanagra.service.query;
+
+import bio.terra.tanagra.service.search.Attribute;
+import bio.terra.tanagra.service.search.EntityVariable;
+import bio.terra.tanagra.service.search.Filter;
+import com.google.auto.value.AutoValue;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+
+/** A query for a dataset of entity instances. */
```

Comment:
Terminology: what does dataset refer to here? Is this using it in the "BigQuery dataset" sense? Or is this like the "dataset" in "PDB"?

Reply:
Dataset, like what gets exported for researchers. Very overloaded, would love a better name.